### ISSUE TITLE
Question about DM0 training stages and Embodied Spatial Scaffolding in the open-source repo

### ISSUE THREAD

**Issue by linjinyu609-art** [dexmal/dexbotic#81]
Hi, thanks for releasing Dexbotic and DM0.

I can find the main DM0 training-related files in the repo, for example:

dexbotic/exp/dm0_exp.py
dexbotic/model/dm0/dm0_arch.py
playground/benchmarks/libero/libero_dm0.py
So it seems that embodied DM0 training / continued training is supported.

However, I could not find an explicit implementation of the Embodied Spatial Scaffolding strategy mentioned in the DM0 paper.

Could you please clarify:

Is the current repo sufficient to train DM0, or mainly for fine-tuning / continued training only?
Which parts of the paper’s three-stage pipeline are already open-sourced?
Is Embodied Spatial Scaffolding (SS) already implemented in this repo?
If not, is there any plan to release that part later?
Thanks.

**Comment by dexmal2025** [comment]
Hi, thanks for your interest on our DM0 and dexbotic!  The Embodied Spatial Scaffolding strategy is adopted during pretraining and post-training stages. The current repo is mainly for fine-tuning with only flow-matching loss, similar to the openpi. In the near future, we plan to opensource the code of hybrid training.
